modules/Movecraft/src/main/resources/localisation/movecraftlang_en.properties

```diff
@@ -137,3 +137,28 @@ Error=ERROR
 Direct\ Control\ -\ Entering=Entering Direct Control Mode
 Direct\ Control\ -\ Leaving=Leaving Direct Control Mode
 You\ must\ be\ piloting\ a\ craft=You must be piloting a craft
+
+Cruising\ -\ Main\ action\ bar\ text=Cruising 
+Cruising\ -\ Enabled\ action\ bar\ text=Enabled
+Cruising\ -\ Disabled\ action\ bar\ text=Disabled
+
+Sign\ -\ ON=ON
+Sign\ -\ OFF=OFF
+Sign\ -\ Ascend=Ascend: 
+Sign\ -\ Cruise=Cruise: 
+Sign\ -\ Descend=Descend: 
+
+Sign\ -\ Contacts=Contacts:
+Sign\ -\ Helm=[helm]
+Sign\ -\ Move=Move:
+Sign\ -\ Name=Name:
+Sign\ -\ Pilot=Pilot:
+Sign\ -\ Relative\ move=RMove:
+Sign\ -\ Release=Release
+Sign\ -\ Remote=Remote Sign
+Sign\ -\ Scuttle=Scuttle
+Sign\ -\ Speed=Speed:
+Sign\ -\ Status=Status:
+Sign\ -\ Status/Fuel\ range=Fuel range:
+Sign\ -\ Subcraft\ rotate=Subcraft Rotate:
+Sign\ -\ Teleport=Teleport:
```

Comment:
I'm not yet sure how I feel on making the sign text available for localization.  Can you move that into a separate PR?